backend/backend/asgi.py

```diff
@@ -8,9 +8,17 @@
 """
 
 import os
-
+from channels.routing import ProtocolTypeRouter, URLRouter
 from django.core.asgi import get_asgi_application
 
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'backend.settings')
+# Initialize Django ASGI application early to ensure the AppRegistry
+# is populated before importing code that may import ORM models.
```

Comment:
Im not a fan of commenting code, comments get obsolete and honestly a basic django call can be googled in 1 sec if one needs to understand what is happenging.